SPECS/xs-openssl.spec

```diff
@@ -24,7 +24,7 @@
 Summary: Utilities from the general purpose cryptography library with TLS implementation
 Name: xs-openssl
 Version: 1.1.1k
-Release: 6.1%{?dist}
+Release: 9%{?dist}
```

Comment:
Should be:
```suggestion
Release: 9.1%{?dist}
```

Reply:
I was unsure if we wanted to add the .1 for the first version, but I forgot to remove it from the changelog so it wasn't even coherent... Fixed.

Reply:
The rationale here is that -9 is the upstream release, so we leave it to them and go with -9.1